base_export_manager/views/ir_exports_view.xml

```diff
@@ -1,84 +1,81 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
-
 <odoo>
 
-<record id="ir_exports_action" model="ir.actions.act_window">
-    <field name="name">Export Profiles</field>
-    <field name="res_model">ir.exports</field>
-    <field name="view_type">form</field>
-    <field name="view_mode">tree,form</field>
-</record>
-
-<menuitem id="ir_exports_menu"
-          parent="base.next_id_2"
-          name="Export Profiles"
-          action="ir_exports_action"/>
+    <record id="ir_exports_action" model="ir.actions.act_window">
```

Comment:
IMO the diff is edited pretty heavily here and I am unable to easily ascertain what is being changed. Possible to change the indentation back please?